### PR TITLE
refactor(parental-leave): review screen to show data non editable by default

### PR DESCRIPTION
**Pull request by jeremybarbet**
## What

- https://app.asana.com/0/1182378413629561/1200214178491343

https://user-images.githubusercontent.com/937328/118646527-d388c000-b7cf-11eb-9ad8-4b909a94b1ec.mov

### CODE REVIEW

File: libs/application/core/src/lib/messages.ts [hunk 1]
```diff
@@ -134,6 +139,16 @@ export const coreMessages = defineMessages({
     description:
       'To continue the application process, you will need to sign in.',
   },
+  radioYes: {
+    id: 'application.system:radio.option.yes',
+    defaultMessage: 'Já',
+    description: 'Yes option value',
+  },
+  radiosNo: {
```
Comment (baering) [inline libs/application/core/src/lib/messages.ts:147]: Very minor, but you have plural `radios` here.

File: libs/application/templates/parental-leave/src/fields/Review/index.tsx [hunk 2]
```diff
@@ -171,231 +148,456 @@ const Review: FC<ReviewScreenProps> = ({
...
-            <Box paddingY={4}>
-              <GridRow>
-                <GridColumn span="6/12">
-                  {editable ? (
-                    <Input
-                      id="payments.bank"
-                      name="payments.bank"
+    <>
+      <ReviewGroup
+        isEditable={editable}
```
Comment (baering) [inline libs/application/templates/parental-leave/src/fields/Review/index.tsx:154]: Here we need to keep in mind that only a primary parent can edit the other parent

File: libs/application/templates/parental-leave/src/fields/Review/index.tsx [hunk 3]
```diff
@@ -411,87 +613,205 @@ const Review: FC<ReviewScreenProps> = ({
...
+              <DataValue
+                label={formatMessage(parentalLeaveFormMessages.employer.email)}
+                value={employerEmail}
+              />
+            )}
+          </GridColumn>
+        </GridRow>
+      </ReviewGroup>
+
+      <ReviewGroup isEditable={editable}>
```
Comment (baering) [inline libs/application/templates/parental-leave/src/fields/Review/index.tsx:749]: I assume this will not be editable so can probably just be `false`

**Review by baering — COMMENTED** [review]
Looks very good 💯  We'll need to keep in mind which parent is applying to know what they can edit. For example only a primary parent can change the other parent. Also if at some point we want to make rights editable that will also just be for primary parent. I'm not sure how we want to have it for personal allowance, if that should be chosen by primary parent like rights. But we'll discuss it later.

File: libs/application/templates/parental-leave/src/fields/Review/index.tsx (hunk 3 above)
Comment (jeremybarbet) [reply libs/application/templates/parental-leave/src/fields/Review/index.tsx:749]: Good point 👍

**Review by baering — APPROVED** [review]

**Pull request merged**